core/src/main/java/com/linecorp/armeria/server/ServiceRequestContextWrapper.java

```diff
@@ -219,4 +219,9 @@ public void mutateAdditionalResponseTrailers(Consumer<HttpHeadersBuilder> mutato
     public ProxiedAddresses proxiedAddresses() {
         return delegate().proxiedAddresses();
     }
+
+    @Override
+    public CompletableFuture<Void> initiateConnectionShutdown(Duration gracePeriod) {
```

Comment:
Would it make sense to rename this to something like `drainPeriod`?

Reply:
I think it has to start with a verb. `drainPeriod` sounds like it doesn't trigger any action but just setting 'drain period'. Any other suggestions?

Reply:
How about `scheduleConnectionShutdown()`?

Reply:
Oh, I should've been more clear, I wanted to comment on the `gracePeriod` argument :) Is current name OK or it should be `drainPeriod` or `drainDuration`?

I don't mind `scheduleConnectionShutdown` if you prefer it. Not sure if it may become a bit confusing because connection shutdown may start instantly, without scheduling it. `startConnectionShutdown` or `requestConnectionShutdown` are other options.